incubator/cloudflare-datadog/values.yaml

```diff
@@ -0,0 +1,29 @@
+# Default values for cloudflare-datadog.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  repository: mozmeao/cloudflare-datadog
+  tag: latest
+  pullPolicy: Always
+resources:
```

Comment:
We started recommending not to specify defaults for resources. This should be a conscious choice of the user.